client/components/checklist/checklist.js

```diff
@@ -5,25 +5,26 @@ import classNames from 'classnames';
 import React, { Children, PureComponent, cloneElement } from 'react';
 import PropTypes from 'prop-types';
 import { isFunction, times } from 'lodash';
-
+import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import ChecklistHeader from './header';
 import TaskPlaceholder from './task-placeholder';
+import Card from 'components/card';
 
-export default class Checklist extends PureComponent {
+class Checklist extends PureComponent {
 	static propTypes = {
 		className: PropTypes.string,
 		phase2: PropTypes.bool,
 		isPlaceholder: PropTypes.bool,
 		onExpandTask: PropTypes.func,
-		progressText: PropTypes.string,
+		showChecklistHeader: PropTypes.bool,
+		checklistFooter: PropTypes.func,
```

Comment:
This should be a node since it can be anything that can be rendered.

```suggestion
		checklistFooter: PropTypes.node,
```

Reply:
Thanks for this. The errors are all gone and it still works :D